Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM hyperledger/iroha2-base:$TAG AS builder
 
 WORKDIR /iroha
 COPY . .
+RUN  rm -f rust-toolchain.toml
```

Comment:
This is such a controversial addition to the repo that I'm inclined to remove and ban it. If we can't agree on having it, we should agree on getting rid of it permanently. If we need to keep this information somewhere, it could be kept away from the main code, just like `hooks`.

Reply:
Left to #2690, including inspection whether the toolchain file relates or not